app/controllers/newsletters/index.js

```diff
@@ -38,7 +38,7 @@ export default class NewslettersIndexController extends Controller {
     const agendas = await meeting.agendas;
     return agendas
       ?.slice()
-      .sort((a1, a2) => a1.serialNumber.localeCompare(a2.serialNumber))
+      .sort((a1, a2) => a1.serialnumber.localeCompare(a2.serialnumber))
```

Comment:
this is sidetracking, should not be capital N. No errors ever came of this? `latestAgenda` could have been incorrect in this view

Reply:
I accidentally found where this has an effect.
right here (screenshot on PROD)
![image](https://github.com/user-attachments/assets/c62738d3-0b26-4e48-85e6-e4c02ea7aefa)

If there is more then 1 agenda version > no modified.
Only modified when there is only agenda A.
This should be fixed after merging this.